app/lib/trade_tariff_backend/search_client.rb

```diff
@@ -2,6 +2,8 @@
 
 module TradeTariffBackend
   class SearchClient < SimpleDelegator
+    SEARCH_SERVER_CONFIG_FILE = Rails.root.join('config/config_for_search_server.yml')
```

Comment:
two things:
- for easy searchability it would be nice to have the "elasticsearch" (o maybe opensearch now 🤔 ) in the name of the file (maybe "config_for_elasticsearch_server.yml", or even just "elasticsearch_server.yml" since we are already under config)
- I guess we should pass the two pars to build the path: `Rails.root.join('config', 'config_for_search_server.yml')`

Reply:
1. I'd actually abstracted the 'search_server' instead of ES or OS precisely because we're using the ES gem with an OS server - I'll put it as ES for now and switch it if we swap to the OS gem